timescaledb.rb

```diff
@@ -10,12 +10,12 @@ class Timescaledb < Formula
 
   depends_on "cmake" => :build
   depends_on "openssl" => :build
-  depends_on "postgresql@16" => :build
+  depends_on "postgresql@17" => :build
```

Comment:
Are there a way to prevent users to use minor versions < 17.2???

Reply:
good point, let me check if we can reinforce it.